### PR TITLE
finished all animations for third block

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/js/q7.js [hunk 1]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
         handler(items_collection[index], current_control, `${index*(-100)}vw`)
     })
 })
+
+
+// Third-block
+
+// Slider elements
+const thirdBlockSlider = document.querySelector(".Third-block__slider")
+const thirdBlockNext = document.querySelector(".Third-block__right-arrow")
```
Comment (StadniukYevhen) [inline src/js/q7.js]: use single quotes

File: src/js/q7.js [hunk 2]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+const thirdBlockPrevious = document.querySelector(".Third-block__left-arrow")
+const thirdBlockItems = document.querySelectorAll(".items")
+const circleSvg = document.querySelectorAll('.new')
+let pageNumber = document.querySelector(".Third-block__counter")
+let pageCounter = 1
+
+// Animation for svg circles
+const currentSlide = (element, index) => element[index]
+const previousSlide = (element, index) => element[index]
```
Comment (StadniukYevhen) [inline src/js/q7.js]: remove dublicate function

File: src/js/q7.js [hunk 3]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+
+function thirdBlockChangeSlide (element, number, counter) {
+    element.style.marginLeft = (-100)*(counter-1) + 'vw'
```
Comment (StadniukYevhen) [inline src/js/q7.js]: use ES6 string

File: src/js/q7.js [hunk 4]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+    number.innerHTML = counter.toString()
```
Comment (StadniukYevhen) [inline src/js/q7.js]: use innerText instead innerHTML

File: src/js/q7.js [hunk 5]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+    const clickedSvg = document.querySelectorAll('.clicked')
+
+    previousElement.classList.contains('current') && previousElement.classList.remove('current')
+    currentElement.classList.add('current')
+    clickedSvg && clickedSvg.forEach(element => {
+        element.classList.remove('clicked')
+    })
+}
+
+function makeVisible (collection) {
```
Comment (StadniukYevhen) [inline src/js/q7.js]: rename collection to array

File: src/js/q7.js [hunk 6]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+    collection.forEach((element) => {
+        element.classList.add('visible')
+    })
+}
+
+function startHeartBeat () {
+    const smallCircles = document.querySelectorAll('.current .cls-2')
+    const bigCircles = document.querySelectorAll('.current .cls-3')
+    makeVisible(smallCircles)
```
Comment (StadniukYevhen) [inline src/js/q7.js]: use one function call

File: src/js/q7.js [hunk 7]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+    const previousItem = previousSlide(thirdBlockItems,pageCounter-2) || thirdBlockItems[2]
+    const currentItem = currentSlide(thirdBlockItems,pageCounter-1)
+    toggleClass(previousItem, currentItem)
+    setTimeout(startHeartBeat, 1)
+})
+
+circleSvg.forEach(element => {
+    element.addEventListener('click', () => {
+        element.classList.add('clicked')
+        console.log(element)
```
Comment (StadniukYevhen) [inline src/js/q7.js]: remove console.log

File: src/js/q7.js [hunk 8]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+    element.style.marginLeft = (-100)*(counter-1) + 'vw'
+    number.innerHTML = counter.toString()
+}
+
+function toggleClass (previousElement, currentElement) {
+    const clickedSvg = document.querySelectorAll('.clicked')
+
+    previousElement.classList.contains('current') && previousElement.classList.remove('current')
+    currentElement.classList.add('current')
+    clickedSvg && clickedSvg.forEach(element => {
```
Comment (StadniukYevhen) [inline src/js/q7.js]: move function to the next line

File: src/js/q7.js [hunk 9]
```diff
@@ -48,3 +48,75 @@ controls_collection.forEach((current_control, index) => {
...
+    pageCounter > 1 ? --pageCounter : pageCounter = 3
+    thirdBlockChangeSlide(thirdBlockSlider, pageNumber, pageCounter)
+    const previousItem = previousSlide(thirdBlockItems,pageCounter) || thirdBlockItems[0]
+    const currentItem = currentSlide(thirdBlockItems,pageCounter-1)
+    toggleClass(previousItem, currentItem)
+    setTimeout(startHeartBeat, 1)
+})
+
+thirdBlockNext.addEventListener('click', () => {
+    pageCounter < 3 ? ++pageCounter : pageCounter = 1
```
Comment (StadniukYevhen) [inline src/js/q7.js]: use the next style for ternary operator:
```
logic
? state
: state
```

File: src/js/q7.js (hunk 1 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 2 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 3 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 8 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 5 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 6 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 7 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

File: src/js/q7.js (hunk 9 above)
Comment (nmelnychenko) [inline src/js/q7.js]: done

**Comment by nmelnychenko** [comment]
All issues are fixed.

**Pull request merged**